client/starwhale/core/runtime/model.py

```diff
@@ -223,6 +224,26 @@ def asdict(self, ignore_keys: t.Optional[t.List[str]] = None) -> t.Dict:
         ...
 
 
+@unique
+class RuntimeRestoreStatus(Enum):
+    restoring = "restoring"
+    success = "success"
+    failed = "failed"
+    undefined = "undefined"
+
+    @classmethod
+    def mark(cls, status: RuntimeRestoreStatus, rootdir: Path) -> None:
+        ensure_file(rootdir / ".runtime_restore_status", status.value, parents=True)
+
+    @classmethod
+    def get(cls, rootdir: Path) -> RuntimeRestoreStatus:
+        fpath = rootdir / ".runtime_restore_status"
+        if not fpath.exists():
+            return cls.undefined
+        else:
+            return cls(fpath.read_text().strip())
```

Comment:
what if the content in the fpath is not in the Enum values

Reply:
add except catch, then make the status undefined.